src/types.ts

```diff
@@ -919,6 +939,10 @@ export interface FunctionConfig {
     }
   >;
   idempotency?: string;
+  batchEvents?: {
+    maxSize: number;
+    timeout: string;
```

Comment:
🤔  should we make a `SecondTimeStr` type here?

Reply:
chatted in person and decided this can be a follow up.
backend will verify on registration anyways.